dot/sync/chain_sync.go

```diff
@@ -192,12 +193,16 @@ func (cs *chainSync) start() {
 		time.Sleep(time.Millisecond * 100)
 	}
 
+	pendingBlockDoneCh := make(chan struct{})
+	cs.pendingBlockDoneCh = pendingBlockDoneCh
+	go cs.pendingBlocks.start(cs.ctx, pendingBlockDoneCh)
```

Comment:
nit

```suggestion
	cs.pendingBlockDoneCh = make(chan struct{})
	go cs.pendingBlocks.start(cs.ctx, cs.pendingBlockDoneCh)
```

Reply:
`cs.pendingBlockDoneCh` is of type `<-chan struct{}` but the parameter to `start` is `chan<- struct{}` so i don't think this works

Reply:
Ah good point! My bad!